src/app/Mood.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package app;
+
+import javafx.scene.control.DatePicker;
+import javafx.scene.control.Slider;
+
+/**
+ *
+ * @author aswecker
+ */
+public class Mood {
+    private int id;
+    private Slider moodType;
+    private DatePicker moodDate;
+    
+    /**
+    * This is the default constructor for the Mood class.
+    * @param moodType
+    * @param moodDate
+    */ 
+    public Mood(Slider moodType, DatePicker moodDate){
```

Comment:
Please remove these params along with the associated fields. The constructor should take ID and name as the parameters.